src/core/plugins/view/root-injects.jsx

```diff
@@ -101,15 +101,20 @@ const wrapRender = (component) => {
 }
 
 
-export const getComponent = (getSystem, getStore, getComponents, componentName, container) => {
+export const getComponent = (getSystem, getStore, getComponents, componentName, container, config) => {
```

Comment:
**suggestion:** since this is a new optional parameter and we aren't updating existing call sites, let's set a sane default!
```suggestion
export const getComponent = (getSystem, getStore, getComponents, componentName, container, config = {}) => {
```